app/scripts/controllers/transactions/index.js

```diff
@@ -2171,7 +2089,7 @@ export default class TransactionController extends EventEmitter {
   _trackSwapsMetrics(txMeta, approvalTxMeta) {
```

Comment:
This method is going to be removed on swaps decouple task(https://github.com/MetaMask/MetaMask-planning/issues/1051)